runtimes/common/src/constants.rs

```diff
@@ -227,3 +227,8 @@ pub mod treasury {
 	const YEARLY_REWARD: Balance = 2_000_000u128 * KILT;
 	pub const INITIAL_PERIOD_REWARD_PER_BLOCK: Balance = YEARLY_REWARD / (BLOCKS_PER_YEAR as Balance);
 }
+
+pub mod proxy {
+	pub const MAX_PROXIES: u16 = 10;
```

Comment:
Is this an arbitrary value of is there a reason we are going with 10 instead of 32 compared to Kusama?

Reply:
I lowered this on request from @Diiaablo95 to match the number of proxy types, but the longer I think about it, I guess we should increase it again because there might be usecases where you want to setup proxies for the same type but for multiple accounts.

Reply:
Yes I suggested to use a number that would reflect the number of proxy types we have. I think Kusama is way bigger and it is also more reasonable to have 30 proxies per account. In our case that is most likely not realistic, and having a lower number would let us test the proxy feature before potentially bumping the number up. Not sure if Kusama went live straight with 30 as the value.

Reply:
Thanks for the explanation. Seems reasonable!